src/goto-programs/initialize_goto_model.cpp

```diff
@@ -117,9 +118,11 @@ bool initialize_goto_model(
 
       if(binaries.empty())
       {
-        if(language_files.final(goto_model.symbol_table))
+        // For now, if a binary is supplied it is expected to provide
+        // __CPROVER_start and related start-of-day machinery.
+        if(language_files.generate_support_functions(goto_model.symbol_table))
         {
-          msg.error() << "CONVERSION ERROR" << messaget::eom;
+          msg.error() << "ENTRY-POINT GENERATION ERROR" << messaget::eom;
```

Comment:
Though most of `generate_support_functions` is related to the entry point, in might be worth calling this `SUPPORT FUNCTION ERROR`. Alternatively, could have the `generate_support_functions` throw structured exceptions for error states?